python/tvm/auto_scheduler/task_scheduler.py

```diff
@@ -82,11 +82,12 @@ def make_search_policies(
     if isinstance(search_policy, str):
         policy_type, model_type = search_policy.split(".")
         if model_type == "xgb":
-            cost_model = XGBModel(num_warmup_sample=len(tasks) * num_measures_per_round)
-            if load_model_file:
-                logger.info("TaskScheduler: Load pretrained model...")
-                cost_model.load(load_model_file)
-            elif load_log_file:
+            cost_model = XGBModel(
+                num_warmup_sample=len(tasks) * num_measures_per_round,
+                model_file=load_model_file,
+            )
+            if load_log_file:
+                logger.info("TaskScheduler: Reload measured states and train the model...")
```

Comment:
`load_model_file` and `load_log_file` are mutually exclusive, because `update_from_file` will retrain a model and overwrite the loaded model.
I think the old code is better.

I don't know why the old code cannot satisfy your need.

Reply:
Oh, the old one is fine. I was just going to add a `self.model_file` for cost model saving after training, this was modified by the way.